admin/client/src/components/FormBuilder/FormBuilder.js

```diff
@@ -373,23 +328,24 @@ export class FormBuilderComponent extends SilverStripeComponent {
    * @return {Array}
    */
   mapActionsToComponents(actions) {
-    const form = this.props.form[this.getFormId()];
-
     return actions.map((action) => {
-      const loading = (form && form.submitting && form.submitAction === action.name);
-      // Events
-      const extraProps = {
-        handleClick: this.handleAction,
-        loading,
-        disabled: loading || action.disabled,
-      };
-
-      // Build child nodes
+      let props = action;
+      props.handleClick = this.handleAction;
```

Comment:
Could move this and the `if (this.state.submittingAction === action.name) {` below to an `else` statement for `if (action.children) {`.
When reading this code block, it comes as very weird to set the props just to watch them disappear immediately after.